packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -166,24 +195,31 @@ export const DrawerPanelContent: React.FunctionComponent<DrawerPanelContentProps
   };
 
   const handleControlMove = (e: MouseEvent | TouchEvent, controlPosition: number) => {
+    const isRTL = window.getComputedStyle(panel.current).getPropertyValue('direction') === 'rtl';
+
     e.stopPropagation();
     if (!isResizing) {
       return;
     }
 
     if (setInitialVals) {
       panelRect = panel.current.getBoundingClientRect();
-      right = panelRect.right;
-      left = panelRect.left;
+      if (isRTL) {
+        start = panelRect.right;
+        end = panelRect.left;
+      } else {
+        end = panelRect.right;
+        start = panelRect.left;
+      }
       bottom = panelRect.bottom;
       setInitialVals = false;
     }
     const mousePos = controlPosition;
     let newSize = 0;
-    if (position === 'right') {
-      newSize = right - mousePos;
-    } else if (position === 'left') {
-      newSize = mousePos - left;
+    if (position === 'end' || position === 'right') {
+      newSize = end - mousePos;
+    } else if (position === 'start' || position === 'left') {
+      newSize = mousePos - start;
```

Comment:
Looks like we need to take into account the isRTL var when reassigning newSize here:

```suggestion
    if (position === 'end' || position === 'right') {
      newSize = isRTL ? mousePos - end : end - mousePos;
    } else if (position === 'start' || position === 'left') {
      newSize = isRTL ? start - mousePos : mousePos - start;
```

Right now when I click and hold the splitter to resize the drawer with RTL enabled, it just snaps to the smallest size and I'm unable to resize the drawer. The above I was able to resize things in both RTL and LTR as expected.